core/include/thread.h

```diff
@@ -11,6 +11,96 @@
  * @ingroup     core
  * @brief       Support for multi-threading
  *
+ * Priorities
+ * ==========
+ *
+ * As RIOT is using a fixed priority @ref core_sched "scheduling algorithm",
+ * threads are scheduled based on their priority. The priority is fixed for
+ * every thread and specified during the threads creation by the `priority`
+ * parameter.
+ *
+ * The lower the priority number, the higher the priority of the thread,
+ * with 0 being the highest possible priority.
+ *
+ * The lowest possible priority is @ref THREAD_PRIORITY_IDLE - 1. The exact value
+ * depends on the platforms architecture, e.g. it would be 30 in 32-bit systems,
+ * and 14 in 16-bit systems.
```

Comment:
Oh, indeed! Well then I stumbled across an outdated bit of documentation, I guess? An, I've deleted the sentence.